### ISSUE TITLE
How to get user id from username ? (App API)

### ISSUE THREAD

**Issue by nivexz** [ping/instagram_private_api#36]
### Before submitting an issue, make sure you have:
- [x] Updated to the lastest version v1.3.6
- [x] Read the [README](https://github.com/ping/instagram_private_api/blob/master/README.md) and [docs](https://instagram-private-api.readthedocs.io/en/latest/)
- [x] [Searched](https://github.com/ping/instagram_private_api/search?type=Issues) the bugtracker for similar issues including **closed** ones
- [x] Reviewed the sample code in [tests](https://github.com/ping/instagram_private_api/tree/master/tests) and [examples](https://github.com/ping/instagram_private_api/tree/master/examples)

### Which client are you using?
- [x] app (``instagram_private_api/``)
- [ ] web (``instagram_web_api/``)

### Purpose of your issue?
- [ ] Bug report (encountered problems/errors)
- [ ] Feature request (request for a new functionality)
- [x] Question
- [ ] Other

---

### Describe your issue

Hi There !
I am using the api in order to take users' story reel, and my purpose is to take it for users I don't have their user id. Those are users who I don't follow for example (and they are not private) but I just want my program to get a username as an input and return the story feed for the specific user.

In other words, the question is how do I get user_id from username, I need that because the function "user_reel_media(user_id, **kwargs)" need user_id which I don't have, I only have his username.

The only way I found is combining the use of the 2 Api-s: the web and the app, and then I can solve it by using the user_info2 **BUT**, Is this the right way to do that ? 

Thank you for your wonderful contribution

**Comment by notcammy** [comment]
```py
user_info= api.username_info(username)
user_id = user_info['user']['pk']
```

**Comment by nivexz** [comment]
@notcammy Thank you!

**Comment by ping** [comment]
Please close your own issue if resolved.
